src/app/index.locale.js

```diff
@@ -1,16 +1,14 @@
-function localeConfig(configuration, localeConf, localeSupported) {
+function localeConfig(configuration, $translateProvider) {
     'ngInject';
 
-    localeConf.basePath = 'res';
-    localeConf.defaultLocale = configuration.defaultLocale;
-    localeConf.sharedDictionary = 'common';
-    localeConf.fileExtension = '.json';
-    localeConf.persistSelection = true;
-    localeConf.cookieName = 'COOKIE_LOCALE_LANG';
-    localeConf.observableAttrs = new RegExp('^data-(?!ng-|i18n)');
-    localeConf.delimiter = '::';
+    // Tell the module what language to use by default
+    $translateProvider.preferredLanguage(configuration.defaultLocale);
```

Comment:
No matter what the `defaultLocale` is in the `app.json`, it will be loaded here. If that language is not created in the `src/i18n` folder, having a resource file `src/i18n/LOCALE.json` then the fallback language will be used.